apycula/attrids.py

```diff
@@ -276,6 +276,485 @@
+        'OPCDPA11':            370,
+        'OPCDDYN18':           371,
+        'OPCDDYN19':           372,
+        'OPCDDYN110':          373,
+        'RSTGENMUX_ADDSUB0':   374,
+        'MASK01_3':            375,
+        'MASK01_1':            376,
+        'MASK01_2':            377,
+        'MASK01_5':            378,
+        'MASK01_6':            379,
+        'MASK01_11':           380,
+        'MASK01_8':            381,
+        'MASK01_13':           382,
+        'MASK01_14':           383,
+        'MASK01_0':            384,
+        'MASK01_7':            385,
+        'MASK01_4':            386,
+        'MASK01_9':            387,
+        'MASK01_10':           388,
+        'MASK01_15':           389,
+        'MASK01_12':           390,
+        'MASK01_19':           391,
+        'MASK01_17':           392,
+        'MASK01_18':           393,
+        'MASK01_16':           394,
+        'MASK01_25':           395,
+        'MASK01_26':           396,
+        'MASK01_29':           397,
+        'MASK01_28':           398,
+        'MASK01_33':           399,
+        'MASK01_34':           400,
+        'MASK01_43':           401,
+        'MASK01_40':           402,
+        'MASK01_42':           403,
+        'MASK01_27':           404,
+        'MASK01_24':           405,
+        'MASK01_31':           406,
+        'MASK01_30':           407,
+        'MASK01_35':           408,
+        'MASK01_32':           409,
+        'MASK01_37':           410,
+        'MASK01_39':           411,
+        'MASK01_36':           412,
+        'MASK01_38':           413,
+        'MASK01_41':           414,
+        'MASK01_46':           415,
+        'MASK01_51':           416,
+        'MASK01_50':           417,
+        'MASK01_53':           418,
+        'MASK01_52':           419,
+        'MASK01_47':           420,
+        'MASK01_44':           421,
+        'MASK01_45':           422,
+        'MASK01_49':           423,
+        'MASK01_48':           424,
+        'MASK01_54':           425,
+        'MASK01_20':           426,
+        'MASK01_22':           427,
+        'MASK01_21':           428,
+        'MASK01_23':           429,
+        'IRASHFEN_0':          430,
+        'IRASHFEN_1':          431,
+        'IRBSHFEN_0':          432,
+        'IRBSHFEN_1':          433,
+        'OPCD_6':              434,
+        'OPCD_3':              435,
+        'OPCD_10':             436,
+        'OPCD_8':              437,
+        'OPCD_7':              438,
+        'OPCD_1':              439,
+        'OPCDDYN_INV_0':       440,
+        'OPCDDYN_INV_1':       441,
+        'OPCDDYN_INV_2':       442,
+        'OPCDDYN_INV_3':       443,
+        'OPCDDYN_INV_4':       444,
+        'OPCDDYN_INV_5':       445,
+        'OPCDDYN_INV_6':       446,
+        'OPCDDYN_INV_7':       447,
+        'OPCDDYN_INV_8':       448,
+        'OPCDDYN_INV_9':       449,
+        'OPCDDYN_INV_10':      450,
```

Comment:
As usual, I'm not super happy with hardcoded constants so anything we can do to document where they came from would be great. Of course even better would be to obtain them programmatically but that's of course not always possible.